wright-filter-wheels/wright_filter_wheels/_yaqd_wright_filter_wheels_continuous.py

```diff
@@ -17,10 +17,13 @@ def __init__(self, name, config, config_filepath):
         super().__init__(name, config, config_filepath)
         self._motornum=config["motor"]
         self._serial_port = aserial.ASerial(config["serial_port"], config["baud_rate"])
-        self.microstep=1
+        self._microstep=config["microstep"]
+        self._units=config["units"]
```

Comment:
We know the units to be "deg", and cannot be anything else, no need to make that a configuration

Any conversions to e.g. OD, are to be done client side, they are the ones with knowledge of how it is used.

Reply:
I added units to config.toml such that `.get_units` creates a definite output.